code/modules/mob/living/silicon/mommi/say.dm

```diff
@@ -8,6 +8,11 @@
 
 	return "states, [text]";
 
+/mob/living/silicon/robot/mommi/whisper(message as text)
+	if (emagged)
```

Comment:
check for KEEPER instead, since it's what makes them unable to talk

Reply:
Righto.